build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -18,34 +18,42 @@
 from typing import Any, Dict, Sequence
 
 __all__ = [
-    "AndroidDeviceInfo", "BenchmarkInfo", "BenchmarkResults",
-    "execute_cmd_and_get_output"
+    "AndroidDeviceInfo", "BenchmarkInfo", "BenchmarkResults", "BenchmarkRun",
+    "execute_cmd_and_get_output", "execute_cmd"
 ]
 
 # A map for IREE driver names. This allows us to normalize driver names like
 # mapping to more friendly ones and detach to keep driver names used in
 # benchmark presentation stable.
-IREE_DRIVER_NAME_MAP = {
+IREE_DRIVERS_TO_PRETTY_NAMES = {
     "iree-dylib": "IREE-Dylib",
     "iree-dylib-sync": "IREE-Dylib-Sync",
     "iree-vmvx": "IREE-VMVX",
     "iree-vmvx-sync": "IREE-VMVX-Sync",
     "iree-vulkan": "IREE-Vulkan",
 }
 
+IREE_PRETTY_NAMES_TO_DRIVERS = {
+    v: k for k, v in IREE_DRIVERS_TO_PRETTY_NAMES.items()
+}
 
-def execute_cmd_and_get_output(args: Sequence[str],
-                               verbose: bool = False,
-                               **kwargs) -> str:
+
+def execute_cmd(args: Sequence[str],
+                verbose: bool = False,
+                **kwargs) -> subprocess.CompletedProcess:
```

Comment:
Docstring for this function should be updated. (Or maybe just drop the docstring..)